backend/README.md

```diff
@@ -35,8 +57,70 @@ Both `dev-env/run-docker-compose.sh` and `dev-env/run-db-and-local-django.sh` de
 
 * Then go to https://console.cloud.google.com/apis/credentials/consent, and add a user test (typicaly your gmail account)
 
-# Testing
+## Testing
 
 In order to ease your testing and debug time, use pytest : `pytest`
 Moreover, you can run the following command to have a complete recap in a html document for each test:
 `pytest --html=report.html --self-contained-html`
+
+## Code Quality
+
+We use several tools to keep the code quality as good, readable and maintainable
+as possible:
+- `isort` automatically sorts import statements
+- `pylint` static code analyzer which looks for errors
+- `flake8` a wrapper around three popular tools for style enforcement
+
+All of them should be automatically triggered by the continuous
+integration system.
+
+You can run them locally before creating a new commit, by running the
+following commands.
+
+**manual installation**
+
+The commands should be run from the root of the Git repository. 
+
+```shell
+# check file by file
+isort file1.py file2.py
+pylint --rcfile=backend/.pylintrc file1.py file2.py
+flake8 --config=backend/.flake8 file1.py file2.py
+
+# check the whole git index (i.e. all files added with git add)
+git diff --name-only --cached | xargs -r isort
+git diff --name-only --cached | xargs -r pylint --rcfile=backend/.pylintrc
+git diff --name-only --cached | xargs -r flake8 --config=backend/.flake8 
+```
+
+**automatic installtion with Docker**
+
+The commands should be run from the backend sub-folder.
+
+```shell
+docker exec tournesol-dev-api isort file1.py file2.py
+docker exec tournesol-dev-api pylint --rcfile=.pylintrc file1.py file2.py
+docker exec tournesol-dev-api flake8 --config=.flake8 file1.py file2.py
+```
```

Comment:
As discussed, I would suggest to add a lint script that could be called to run these tools in backend:
```
./scripts/lint.sh
```
or
```
docker exec tournesol-dev-api ./scripts/lint.sh
````

Or perhaps 2 different scripts `lint` and `lint-fix` similarly to the frontend scripts?

Reply:
I added `lint.sh`, and I plan to add `lint-fix` with the next pull request with `isort`.